conda/conda-build/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "legate-core" %}
+{% if package_version is defined %}
+    {% set version = package_version %}
+# The tag must be in one of the two formats (where N is a digit):
+# vNN.NN.NN
+# vNN.NN.NN.dev
+{% elif 'dev' in environ.get('GIT_DESCRIBE_TAG', '') %}
+    {% set version = (environ.get('GIT_DESCRIBE_TAG', '') ~ environ.get('GIT_DESCRIBE_NUMBER', '')).lstrip('v') %}
+{% else %}
+    {% set version = environ.get('GIT_DESCRIBE_TAG', '').lstrip('v') %}
```

Comment:
Either this conditional should have a default statement to handle if it's not running under a git repo, or the environ.get() in the fallback should have a default value.

Reply:
Hmm, I think that if we are running in one of these situations, we should provide package version through `package_version`. I don't think that there is really a good default. Let me know what do you think.

Reply:
hrm, agreed. We could just set it to 0.0.0?

Reply:
We could do that, but I would personally prefer an error message that would remind me to set `package_version`. WDYT? Unfortunately, for cunumeric we don't have a choice since we need a default to avoid errors in the initial pass where the environment vars are not defined yet. That's why we need a "fake" default there, but it also can prevent error messages.

Reply:
I tried to remove the defaults, but that does not work:

```
TypeError: argument of type 'NoneType' is not iterable
```

So it seems that without the default, the output of `environ.get` ends up being `None` in the first pass. So we do have to keep the default, even if it is simply `''`. I will elevate the default into a separate variable.

Reply:
Sounds good. Other than that LGTM.

Reply:
Should be fixed. Take a look and confirm if ready for merging.